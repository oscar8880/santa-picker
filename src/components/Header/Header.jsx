import styles from './Header.module.scss';
import React from 'react';

const Header = () => {
  return <header className={styles.Header}>Bla</header>;
};

export default Header;
