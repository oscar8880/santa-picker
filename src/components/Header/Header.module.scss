@import "../../scss/functions";
@import "../../scss/variables";

.Header {
  background-color: $primary-colour;
  background-image: url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8'%3F%3E%3Csvg width='106px' height='96px' viewBox='0 0 106 96' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Ctitle%3Eargyle%3C/title%3E%3Cg id='Page-1' stroke='none' stroke-width='1' fill='none' fill-rule='evenodd' opacity='0.151971726'%3E%3Cimage id='argyle' x='0' y='0' width='106' height='96' xlink:href='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGoAAABgCAYAAAD1uufxAAAAAXNSR0IArs4c6QAAFA9JREFUeAHdXU1sXcUVfnFiEheS4AcEYiQceAkJYAkibESQkkhdJMuwQcqyZgeVurArdQFLZ1GpyRIkKpEuWFCxiRddJFIraiMFYSOC5ACBmJ9KOFEAO8KQZ4Ih/b7Lnae58+bMPXN/rICl55k7c+b8zJmfc87Mfa+xb9++ZuM39PfMM8/08fMbEqlBeXq++eabVt1C7dix41XSaLVaR+um9eGHHw7wUzcdI4uRrU5677///lDPzz//PFDnCNy7d+/ALbfcMkpBfvrppyN8rkuonp6exvXr15v81CkTcUOWRKb169eP7ty581BdMpEWddRDAu+9997+uggtLi5eXF1dHSf+3t7el69evZrk66C3e/fuIYN3bm6utpVidnZ2YsOGDS+R1o0bN8Z//PHHU4Zu1em5c+dGiDNRFNK+ukY6BOr79NNPT5DYJ598MgXB+pmv+i8d5Z29CSO+r679FzIMXLx4cZIyGNmqlof4yL/pr3WYtkdSIu35+fkzqKiDZq04161b19i1a1cy8lxCGBwzv1aZsA9ySU0Gn5lRlK/vkUceqWy54AwN7ROhOrez856ffPJJ0XIN1eXhdevJM/dB6e/BBx/0DhYJPlSe6qKzQmSorqysDHF0lv2jQN9+++2XWB7EZW5mZuZUSGgtD+Q3ZLmyrgqZyCt4fvXRRx8NGUPvVDEAyS90kZk065vN5h67U+6+++7er7/++opdFpu/du3aK1huVj/66KNkb/K1B91t/f39g0tLS3O+em0ZZy428y0h+O3bt/dduXJlKQSTV/fAAw8cRQcuw/Q/I8Fu3rz5qYWFhf0wmJL9S4LLK3/ooYeGYOlts+EyM4oVELpVdlTQZIUSnrYJufnPP//8BM11tzzmmaP8+++/vzevDc31srOX5vjjjz/+cojWyMjI08YVCcGF6jibqAMXpktRBKCD5QLGPENRT8DkXwi1wYhpQJmlTPWHH364SyCJZgysDwd4HX3jjTfavjpTltY/YZ6LpJJR5FVUWSf4448/ntEwmafMEA7OEM6UEIxdV9YJ1vKqld3mzeS5krHvzbOdehVFgCJO8ODg4LVYL31oaKh1//33H7eZ0uRt51YDT5giTjB5I49aGoRjH7AvYtoQ1ji3vnaiogAc5QSTOTq3sPTEzdbHADpvnk5dTGdw5NGh9eELlcU6weQJvA2QxxBet870AQyQJMzk1vuebefWVx9SVGN5eVltrkOg09gID/uI5JXddtttx7777ruJPDjWc7Mts4fGmOswVEZhHIiWa4hfhMueBq9JMDoEZ2T66quvhkNwQUWhodoJxtp6woykEEFfHUcsZuPrvjq3rAoH9rHHHvPuAy4tdPZk0T2HfcE+cXH6nl3n1gdjh5B89UkZQkuTmDFi/VpVcDZJVlEsDzdLaIky2aEiSY68GZW027Nnj2iuc7OtKnRC59WKPXbxrJ0JXQ09BaFwGXmoKkjNvgkZS2nf5u63KkVJTjCFgZk8duutt37p6YvoIuw9CzgSed7ncGudWy1RyQkmHfIAZZWKZBg+EHKawxI65lM8Z5PPuTVt7bQrhGRX2nmEYPqwEWec2E2bNv0bS+IZhIpes2GL5rm83nHHHasIw/weoaWzNh7OgCKWno3Dzd9zzz1doSWM/jEoa/qtt94658IXef7ggw9WEVoaROzuTwgt/d3GwdkGmTfbZVJeNaPYWHCClxA2+aOEvEj5Z5999jpGWmZAcJTHOLdauoITvEIetDg0cAiXPQuFZEJdIefWh1NlTFgN20UtOwtHdJZByqpnk2EC4a42Aq2lAsMGV0wKo+gAlCeeLri41Etf2rAXHvfyli1b9tx3332Ny5cvL7sIq3rmiMNMGjx48ODKpUuXMqOxKhrEg87qxRK0ctddd/Xv379/lUtVlfhtXNyn4DPuQ1RlCT7abrsuLx+rqGQJwjI4hQ33n+6elUcspn7jxo3bcKb1Vxy5REUFYmgY2Ha73UTH/QXp2ToHHwc4+w7HQP8B7V5DX5Oq9yiDjBYRPo2ijqDBk5cyCIq9ahG0art4Y3gAjRHQamsDr6ZdbMo+Ay1ati/Eto1WFAQ6giXpRaSxtKLgiR/7xyRGoOjDRSEMAIPGMC29NZLpGOgcCLDjrYo1JjpI4BvM17kJcz3XHAp2GKogg7jeYmwANoYsjSKt3+TiVc8ojLomPwaB5ASb+jIpzXFbSdjw+/gpg9PXljghU8fykpxgX9vYMs5WW0luf+bhUysK5vHfIFhmGSoTxQ4x5p7GQqgBrOulju199IgTuDMyubR97YqUuTFK9iX6VBVdJz2VooBwCFZYE6Mi42+wA33hniKCmDacTa5zi7IFCMbIRNQBnsHpS8F7PzsL++CsXS84wTZIdN7n3ILuFLYPXvNW7Vcq8xzIXoFgk0CeCeuQY5izA7i2XJkJzSAlOjBjumKArKJsBZ9h8FBJaAez6SgGwAzwdfEOl2Bz2ZtYtjax5D2F566lGzxwSR8FD7mRENWMQkdN4rzopE3cykedBFvturIceRgUXQIREPRnwMf5rkYFC4gLOKd9zclDVdehQye37FPwIfVrhjXVjMLIC45iLhcIonaNzAylnAcwzM12bwiMS2CoPqYuDxedYKwUpehRJmwZ9AMzK4TNJ/i4YD9L+eCM4maLj+bcX30SLDGiPbkFPyP8SHjyytF2SNu+7PmX5uSW/GIGk6exEO+iorAnNXmZEKNCdfWrzHVojjzeZQgxauowAi+CtyNY271LpIHzpWzDtsChOj+ji0DeivyxnXstWcID2Db6+gD7XIIRFQUtj/3www+L2Ogylp6EiOWhk+BQu5iRi05egmBzGIFcUqL+2AZt54FDvaSFToJDxLUnt8QBfubR1/PoczG0JCoK7fugpGMhZty6Ik4wmMw4ty5O3zM2Yd7tjp5RbAOZTvtwSmVFnGDOJtu5lXDb5Xl9XTiEZBOx8xytMQFb3p1z/SYb382Qjw0t8eQWy5jqppNWvq4ZxY0NH5UT5iNCBrVOMGdTGSWBz5amQwDTT1gfv5oy8qiVyefcamgYGPY9PploCeu6FIWyCWy6hYUiUu116CLXkonf/IHPJvad3NASYI4S1rQrkiJYq+qT0LVkDd2077suo2YURW1iraRzqXLCAoRzneCQcxvAm6kCn7RI+6CIrhFoANOZ1ExhTXF0Cjy5TnDIudUSNH0PmTJuUUZRQDYGhlS3O/MIh65Dc7OtKqCLgcUzK9ECRF0LMLkhmjx5WB+6Dk2Z8q4la2ikMCeAL7NSVG5M2MzgOhlWjO4L9hx5Wr/Jxncz5HmH0XcSTKOIvmRdPCYzCqOOZ02qNTiGEZ8TzJFXh5LAP6PhHZOdeZbF8KuB9TnBlEnr3GpoGBjqBJ9kb00UheWOS15mTTTAZVPXCY5xbmNog3+GljpLIPMsi8GhhXWd4BjnVkuDcFDSEciQOMH8ip0WzkV4LlPJ3uQyYjvBRZxbF5/0jE14GrPoAORJZhbyI5BJFf6ScErlthNMmWKdWwmvWw7c/0BQl7OqtR5XmHjWdAWC/ssFrOrZXIeu41qy4RHLzyrk2AAF3Yt0EM8LkKmyYxFDx6TmOjRPbkFTdS3ZtNWmkKGNQcHzub3r8FYco7ZL0N6iFkERONy/nrHvQRTBoWkDJSX7EoRcoaCaNkVhGLGo04AwfEGm1npcyt9Wt0AkiOXhd6BT2y1UI1SqICqpdlrYP3hCW8tsMvIwhSxLPejACRAsHDKyEQbyvLM+FaivpApytCDPIX5CTnAlxICEL8MhqXXWUjfUEVa8njPYdLn81faHZW8OI6/Bd4VqIwLEEIpnTTzGmMdykXEYq6aLlYgviTdwZz1zOaZqOsA3hhk10wMlTUFAvrxVi3kOQp2rwvzCDNCrZQSCf5rixD/PD4T7Mi2rvO8ow9mzZ5M9nSmXpsqJAKHRCYyiycSPAqFxfObrIIYbr9M23gsXLtQyq8D/IgUytNCZkywzz1WmiEJk+gq+YS1uQKqTF8l7rSEkLEHes6lfwxmUpFjpbKqOMyibh2RGmQJsWsdpCprnsqm0J+EdpMyILEMH+0Qf+Bb3Iy5/xmQvQ8e0lXhPDQsDVioFv03qwkaSURQq2tivxHN7u2Feni8RSF/yBEYaVb2gDUVQSaF9bxNkKvRFJa6M5Jm8+/5oWFBmX11sWaqDjEwZRWFdPwFiTQCWjQK38QJ2cC/CAZv6gokkKGcKOmgIe1NmH7ThWQeYSq5D5/GcypzpYJsXTR4ytRg2oi5s+IyisKfwxTGe75T6fnIcbySmq03IzXME0sR1y2OeMaBGwDONhmDnAIbfolJqSTfmeIg/ykTZQzB5dZBpFGGjKerChq3DmFB/iTA6OLliBuY6xxM2czdTXvtNL5RJ800ssbJlZpTduOjyRweQI0vzRzjX1NW0IwxmSPQtnyJtSEszmwjHP8pU1AkO9bmoKAg1YRyuX1hQ/e84gipoABVxgmnJQahDWhoGDm0OoO1+86xJsVdEy1TECWZfs88lnkRFgcFx+Ax80Ut9e8d1biWibnmME4wRm1xLBn9nXDx5z2hzGvIcJo48WFNfdMbHOMHsYxg9DH9lDAjDA1NRUWjEa7aLQPIHu4GUB/yCZI5LbUw5aDToSJrnUIpZMYx9YI70QnC+OrThdegZ4vDVu2XkqahMbKflEQOH7/by+viUy4N5DhoT1DQA+0Ew15LBTx/QWjR4o1Nuwu7rkz4koNEP2MJnTZxN+Gyi0nz47TJE/GfKygTDQnTGDa20n3kHXRys4owiEjbUKImOXhmBSAudp3KC2cFQVNAcJz7pj22JQ6o35SHn1sDkpZRJ4wSn/SwqiXSCijKMYKk4hY/kBOc6twZPXhpyKDEQBuBfPJeHQ1tPXJBJ9K1CvGhpEC7kBEMmOuKnNPhUisLIYFTaa5FonFsNI4ThCJScYFhF3GwrO/shLnSS13KMMcfzZKNMkhOMuuP4iPuSjVulKFgkJyFY8mac3Rj59vnz53P3L6dN8PHtt9/mpppZ2jib0Kj0tWSbMGRKjiYwALpWCvJgw5bNp5dQMzJxhYKcvD5+QoNfpSgiQmfxa3UyxGKcWw0zhOEI9JjEbQ4WLQ4tHHB2hZ+qnE02H64TzL6EslRKIp6g1WcT8uRr/e6+Or+jzyNLUsSZXOfXBpU5s1LPKCNcembVLOrcGjx5KZ1gzK7gWVMeDm09z7NIyzOTtShUcDyfw8rEsybvfh9CEq0oIOsHsaNFHcEQM3YdaHBPpGOqjiLY7WPzpFW3TMTPvgNv0XHKaEVhefgzPPZD/Pav2M6IgU/f3OO15Kh3bmNoGFjsVTwGOax9q9C0i02Jn30HmaLejSYd1ReC2AyB0Bx8kBWMwP/hrYzoMI6NK5TH4dk2WGRLuGp2DkvFlhBs2Tp8fegXoLWAm7yLdX4DJmjshSzLcKZfQz7zQ155MsQaE+qzpjzC2npYR6rQkhafD0571uRrW6SMMsWeWUUtfa5zG/NrLjEC0Toy8DTXJSfYwJRJXXPcpl0Gr9sW36feuaxCmSQn2G1nnmMU1eXcYvmbqFpZsLx4LTlzFcDnBBsByqaucwvaR/hV12Xx2u35k00IGGRuI/ucYLuNm1cryufcIuA4imn8qou0zDN/nghr+Es2Do7AOkxndzaRJmifhLIqiykSJwb0acjwLPP2n+sE23VuXqso7yknv6wem2KbI8ZFXOSZswmKX/C990vTlg5pEby+NsRlriXb9aQNHubJi11eNM++gUXZxjFQV2SF9DHTVAaZypiAKX6mbh9D0xGMN0Lwzv6laSPB0Pm8GWSiyY6Xt3MHeu6MosY1ApXdhDV+GZ1gnrpKna8tJw6NTGX9Kk2fkA/NrMpVFE85lR3wjqazfbjYIfhixo5V5IMxZVVE66VryYaGSWdnZyeKKitt947BFUo116GDioo5uYUTfBKdrToEc5l+9913n8Oe0XkTw623n2lY8PTVLovJsy1npuYPI30Gyip0xRs/V/5f9omGTmquB28WhxQVdXLLnydCZ49oprvNPEcewzcxPwVU5vQ1pi1+MvB1dGIrdqUgPJZXdIn+J5vSlUI0lkRFuc6t3bm+PNdaWEuHi4x2hHCe145y0uYILOIE+8xxnyx2GXh7EQZM7h0Luw3zsIaf1eyDpl06q8RDWMnqW/NQkWFYm2JQRF+HXutQkVYWGy51dbpODLwzyufc2sjy8viNKdUegrBK9LmMoc0RGOMEF5lNhhZTLa9a2W3cdl5ygn2K8jqCNjJFvt+Obfng0zBN18jxwUplXFo0TrDk3Ep4pXIEUjOhLRcu/c6O5Hsu3Drts+QEdykKG6H4rpGWWLPZ3AmLcSxk2iJM88Ltt9+uMslDdDXXoWNmnkRreHj4GEJBo3S6pT8sx8dxPHNQqteW+97UzJxH0fGanp7+QotQguOZDn8pEz+bIJ5ZYSniz9R1/YSEhFMq5xK4fft28RcI6Ny++eabV6T22nLwukqesbSJZ1Zbt24dhKX4shanBEdad9555xbIttnAZIyJsteSDdK1TmlYSNehy15LXmtZDD3KZF+H7szjGOfWINOk6brdAa36CIGIOat8bkGMc9thUJGBZZa5T+7KqECRC0KZ4CJ1nGCjqCjnNpeKBcB1OzU5G3QEqz5CMKR8jqyvzMCXSeEjHTXRdcqEOxel91ofP7YTnCgq1rn1IZXKMDLGoZwktIRfdh7HzC29hvtocQTSBDd1Zc1xg8eXUgbcr3iedfihlYtQ3LgPrmxZOqt+kYmjnethnX8w1W8Q/44dOyo9ZHR5phy8uMkw1hrIlMwi7COJbC4vVT4nK1JVv5NUJWNlcNEl+K3JRHn+D0L4FaZ9bE8EAAAAAElFTkSuQmCC'%3E%3C/image%3E%3C/g%3E%3C/svg%3E");
  color: $white;
  font-family: $font-family-headings;
  height: 120px;
}

.Header__Wrapper {
  align-items: center;
  display: flex;
  flex-direction: row;
  height: 100%;
  justify-content: center;
  margin: 0 auto;
  max-width: 948px;
  @media (min-width: $breakpoint-lg) {
    justify-content: flex-start;
    padding-left: $spacer-6;
  }
}

.Header__Text {
  padding-left: $spacer-2;
}
