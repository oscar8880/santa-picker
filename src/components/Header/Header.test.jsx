import React from 'react';
import { render } from '@testing-library/react';

import Header from './Header';

test('matches snapshot', async () => {
  const { asFragment } = render(<Header />);

  expect(asFragment()).toMatchSnapshot();
});
