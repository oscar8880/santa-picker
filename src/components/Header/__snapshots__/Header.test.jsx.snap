// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="Header"
  >
    <div
      class="Header__Wrapper"
    >
      <div
        class="Header__Banner"
      >
        <a
          class="Action Action__Appearance--none"
          href="/"
        >
          <svg
            viewBox="0 0 512 512"
            width="65"
          >
            <circle
              cx="399.61"
              cy="57.832"
              fill="#eeeded"
              r="48.46"
            />
            <path
              d="M414.745 83.719c-26.763 0-48.459-21.696-48.459-48.459 0-5.089.789-9.994 2.244-14.602-10.621 8.889-17.381 22.24-17.381 37.174 0 26.763 21.696 48.459 48.459 48.459 21.674 0 40.02-14.23 46.215-33.856-8.414 7.041-19.249 11.284-31.078 11.284z"
              opacity="0.1"
            />
            <circle
              cx="74.5"
              cy="253.86"
              fill="#f7c2ad"
              r="46.861"
            />
            <path
              d="M74.495 284.117c-16.685 0-30.259-13.574-30.259-30.259s13.574-30.259 30.259-30.259 30.26 13.574 30.26 30.259-13.575 30.259-30.26 30.259zm0-45.518c-8.414 0-15.259 6.845-15.259 15.259s6.845 15.259 15.259 15.259 15.26-6.845 15.26-15.259-6.846-15.259-15.26-15.259z"
              fill="#e86478"
              opacity="0.2"
            />
            <path
              d="M74.495 206.997c-3.109 0-6.146.31-9.086.888-3.544 14.747-5.435 30.138-5.435 45.973s1.891 31.225 5.435 45.973c2.94.578 5.977.888 9.086.888 25.881 0 46.861-20.98 46.861-46.861s-20.98-46.861-46.861-46.861z"
              opacity="0.1"
            />
            <circle
              cx="437.5"
              cy="253.86"
              fill="#f7c2ad"
              r="46.861"
            />
            <path
              d="M437.505 284.117c-16.685 0-30.26-13.574-30.26-30.259s13.575-30.259 30.26-30.259 30.259 13.574 30.259 30.259-13.574 30.259-30.259 30.259zm0-45.518c-8.414 0-15.26 6.845-15.26 15.259s6.846 15.259 15.26 15.259 15.259-6.845 15.259-15.259-6.845-15.259-15.259-15.259z"
              fill="#e86478"
              opacity="0.2"
            />
            <path
              d="M437.505 206.997c3.109 0 6.146.31 9.086.888 3.544 14.747 5.435 30.138 5.435 45.973s-1.891 31.225-5.435 45.973c-2.94.578-5.977.888-9.086.888-25.881 0-46.861-20.98-46.861-46.861s20.98-46.861 46.861-46.861z"
              opacity="0.1"
            />
            <circle
              cx="256"
              cy="253.86"
              fill="#f7c2ad"
              r="179.38"
            />
            <g
              fill="#eeeded"
            >
              <path
                d="M256 71.57c-100.675 0-182.288 81.613-182.288 182.288S155.325 436.147 256 436.147s182.289-81.613 182.289-182.289C438.289 153.183 356.675 71.57 256 71.57zm0 334.001c-83.789 0-151.713-67.924-151.713-151.713S172.211 102.145 256 102.145s151.713 67.924 151.713 151.713S339.789 405.571 256 405.571z"
              />
              <path
                d="M367.206 263.551a92.807 92.807 0 00-21.427 1.765c-10.566 2.129-21.613-.557-29.791-7.577-16.12-13.837-37.078-22.199-59.989-22.199s-43.868 8.362-59.989 22.199c-8.178 7.02-19.225 9.706-29.791 7.577a92.807 92.807 0 00-21.427-1.765c-47.51 1.576-86.416 39.691-88.893 87.163-2.772 53.138 39.493 97.092 92.024 97.092 1.197 0 2.388-.023 3.574-.069a23.408 23.408 0 0121.895 12.991C188.458 491.112 219.788 512 256 512s67.542-20.888 82.606-51.272a23.411 23.411 0 0121.895-12.991c1.186.046 2.377.069 3.574.069 52.531 0 94.797-43.955 92.024-97.092-2.477-47.472-41.383-85.588-88.893-87.163z"
              />
            </g>
            <path
              d="M376.897 428.351a94.24 94.24 0 01-3.574-.068 23.408 23.408 0 00-21.895 12.991 92.06 92.06 0 01-7.425 12.392 23.426 23.426 0 0116.497-5.929c1.186.046 2.377.069 3.574.069 36.35 0 67.78-21.05 82.778-51.622-16.9 19.691-41.97 32.167-69.955 32.167zm-108.075 64.195c-36.212 0-67.542-20.888-82.606-51.272a23.411 23.411 0 00-21.895-12.991 94.24 94.24 0 01-3.574.068c-52.531 0-94.797-43.954-92.024-97.092.661-12.665 3.936-24.652 9.254-35.469-12.833 14.952-20.986 34.029-22.076 54.923-2.772 53.137 39.493 97.092 92.024 97.092 1.197 0 2.388-.023 3.574-.069a23.41 23.41 0 0121.895 12.991C188.458 491.112 219.788 512 256 512s67.542-20.888 82.606-51.272c.077-.155.166-.3.245-.453-16.9 19.748-41.998 32.271-70.029 32.271z"
              opacity="0.1"
            />
            <path
              d="M427.72 156.448C427.72 70.044 350.838 0 256 0S84.28 70.044 84.28 156.448h343.44z"
              fill="#e86478"
            />
            <path
              d="M426.867 140.796H116.825c0-54.052 30.088-101.701 75.837-129.806C129.172 33.958 84.28 90.425 84.28 156.448h343.44c0-5.283-.292-10.503-.853-15.652z"
              opacity="0.1"
            />
            <path
              d="M393.859 134.59H118.141a182.084 182.084 0 00-34.534 59.903c1.602.263 3.244.406 4.921.406h334.946c1.676 0 3.319-.143 4.921-.406a182.107 182.107 0 00-34.536-59.903z"
              opacity="0.1"
            />
            <path
              d="M423.473 179.818H88.527c-16.654 0-30.155-13.501-30.155-30.155 0-16.654 13.501-30.155 30.155-30.155h334.946c16.654 0 30.155 13.501 30.155 30.155 0 16.654-13.501 30.155-30.155 30.155z"
              fill="#eeeded"
            />
            <path
              d="M437.436 165.806H102.489c-16.654 0-30.155-13.501-30.155-30.155 0-4.696 1.074-9.14 2.989-13.103-10.036 4.896-16.951 15.196-16.951 27.114 0 16.654 13.501 30.155 30.155 30.155h334.946c11.958 0 22.29-6.961 27.166-17.052a30.022 30.022 0 01-13.203 3.041z"
              opacity="0.1"
            />
            <path
              d="M177.248 227.982a7.5 7.5 0 01-7.5-7.5V207.41a7.5 7.5 0 0115 0v13.073a7.5 7.5 0 01-7.5 7.499zm157.504 0a7.5 7.5 0 01-7.5-7.5V207.41a7.5 7.5 0 0115 0v13.073a7.5 7.5 0 01-7.5 7.499zM256 294.747a35.036 35.036 0 01-26.417-12.032 7.5 7.5 0 0111.308-9.855c3.814 4.376 9.321 6.886 15.108 6.886s11.294-2.51 15.108-6.886a7.5 7.5 0 0111.309 9.855A35.036 35.036 0 01256 294.747z"
              fill="#485458"
            />
          </svg>
        </a>
        <div
          class="Header__Text"
        >
          <a
            class="Action Action__Appearance--none"
            href="/"
          >
            <p
              class="Text Text__banner"
            >
              Santa Picker
            </p>
          </a>
        </div>
      </div>
      <div
        class="Header__Authentication"
      >
        <button
          class="Action Action__Appearance--none"
          type="button"
        >
          <svg
            viewBox="0 0 512 512"
            width="35"
          >
            <path
              d="M218.668 240H16c-8.832 0-16-7.168-16-16s7.168-16 16-16h202.668c8.832 0 16 7.168 16 16s-7.168 16-16 16zm0 0"
              data-original="#000000"
              fill="#fff"
              xmlns="http://www.w3.org/2000/svg"
            />
            <path
              d="M138.668 320a15.89 15.89 0 01-11.309-4.691c-6.25-6.254-6.25-16.387 0-22.637l68.696-68.692-68.696-68.671c-6.25-6.254-6.25-16.387 0-22.637s16.383-6.25 22.637 0l80 80c6.25 6.25 6.25 16.383 0 22.637l-80 80A15.975 15.975 0 01138.668 320zm202.664 192c-23.531 0-42.664-19.137-42.664-42.668v-384c0-18.238 11.605-34.516 28.883-40.512L455.723 2.09C484.395-6.7 512 14.57 512 42.668v384c0 18.219-11.605 34.473-28.863 40.488L354.922 509.91c-4.672 1.45-9 2.09-13.59 2.09zm128-480c-1.387 0-2.559.172-3.816.555L337.879 75.113c-4.184 1.453-7.211 5.676-7.211 10.219v384c0 7.277 7.89 12.184 14.484 10.113l127.637-42.558c4.16-1.453 7.211-5.676 7.211-10.219v-384C480 36.801 475.223 32 469.332 32zm0 0"
              data-original="#000000"
              fill="#fff"
              xmlns="http://www.w3.org/2000/svg"
            />
            <path
              d="M186.668 106.668c-8.832 0-16-7.168-16-16v-32c0-32.363 26.3-58.668 58.664-58.668h240c8.832 0 16 7.168 16 16s-7.168 16-16 16h-240c-14.7 0-26.664 11.969-26.664 26.668v32c0 8.832-7.168 16-16 16zm128 341.332h-85.336c-32.363 0-58.664-26.305-58.664-58.668v-32c0-8.832 7.168-16 16-16s16 7.168 16 16v32c0 14.7 11.965 26.668 26.664 26.668h85.336c8.832 0 16 7.168 16 16s-7.168 16-16 16zm0 0"
              data-original="#000000"
              fill="#fff"
              xmlns="http://www.w3.org/2000/svg"
            />
          </svg>
        </button>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
